python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -628,6 +661,25 @@ async def offset_xy(self, x, y, relative=False, persistent=False):
             )
             await self.offset_azel(az, el, relative)
 
+    async def reset_offsets(self):
+        """Reset all offsets.
```

Comment:
maybe say "Reset both persistent and relative offsets" 
Then say which methods to use to remove just one, or just the other....
Actually, you could add flags here such that it also has the capability to reset only persistent or only relative?